python/ray/rllib/dqn/dqn.py

```diff
@@ -12,8 +12,7 @@
 import ray
 from ray.rllib.common import Agent, TrainingResult
 from ray.rllib.dqn import logger, models
-from ray.rllib.dqn.common.atari_wrappers_deprecated \
-    import wrap_dqn, ScaledFloatFrame
+from ray.rllib.dqn.common.atari_wrappers_deprecated import wrap_dqn
```

Comment:
I understand the "deprecated" part is vestigal from OpenAI's codebase, but should we keep it?

Reply:
Renamed